data/rendering-apps.yml

```diff
@@ -0,0 +1,489 @@
+# Generated by running this in the content store console:
+# y ContentItem.distinct(:nameocument_type).map { |document_type| { name: document_type, apps: ContentItem.where(document_type: d).distinct(:rendering_app)
```

Comment:
`nameocument_type`, guessing that's a typo?

Reply:
The `y x` will output `x` as YAML!

You're right on the rest, I did some super dodgy copy pasting! PR upcoming.

Reply:
Yeah, the `y` command is super useful.

Reply:
TIL! 🎉